MonoGame.Framework/Audio/Xact/SoundBank.cs

```diff
@@ -302,11 +302,11 @@ public void PlayCue(string name, AudioListener listener, AudioEmitter emitter)
 
             XactSound[] sounds;
             if (!_sounds.TryGetValue(name, out sounds))
-                throw new InvalidOperationException();
+                throw new ArgumentException(string.Format("There is no XactSound with passed name '{0}'.", name), "name");
 
             float [] probs;
             if (!_probabilities.TryGetValue (name, out probs))
-                throw new ArgumentException ();
+                throw new ArgumentException (string.Format("There are no probabilities with passed name '{0}'.", name), "name");
```

Comment:
These error messages are misleading.  Use... `"No probabilities found for cue name '{0}'."`.